lib/Widgets/DatePicker.vala

```diff
@@ -126,16 +123,16 @@ namespace Granite.Widgets {
             Object (format: format);
         }
 
-        private void on_icon_press (EntryIconPosition position) {
+        private void on_icon_press (Gtk.EntryIconPosition position) {
             Gdk.Rectangle rect = Gdk.Rectangle ();
-            position_dropdown (out rect);
+            position_dropdown (ref rect);
             popover.pointing_to = rect;
             popover.position = Gtk.PositionType.BOTTOM;
             popover.show_all ();
             calendar.grab_focus ();
         }
 
-        protected virtual void position_dropdown (out Gdk.Rectangle rect) {
+        protected virtual void position_dropdown (ref Gdk.Rectangle rect) {
```

Comment:
Leave the `out` parameter here and initialize the Rectangle here: `rect = Gdk.Rectangle ()`